src/CardContainer/CardContainer.js

```diff
@@ -21,6 +23,17 @@ const CardContainer = ({ movies }) => {
       <section >
         <Header />
         <section className="CardContainer">
+        { !cards.length && (
```

Comment:
Love this UI/UX choice to navigate users to the home page. Also very readable for developers to see why and how we are rendering this display.